test/integ/sso/test_unit_mfa_cache.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2012-2020 Snowflake Computing Inc. All right reserved.
+#
+
+import json
+import os
+
+import pytest
+from mock import Mock, patch
+
+import snowflake.connector
+from snowflake.connector.errors import DatabaseError
+
+try:
+    from snowflake.connector.compat import IS_MACOS  # NOQA
+except ImportError:
+    IS_MACOS = None
```

Comment:
Okay, I guess we need the fallback for old driver test, otherwise it would fail. And I found similar fallback in test_unit_sso_connection.py, which I think is for the same goal. Maybe we can use another way to avoid explicitly fallback in every file later.